src/demo/movement.rs

```diff
@@ -3,6 +3,7 @@
 //!
 //! In our case, the character controller has the following logic:
 //! - Set [`MovementController`] intent based on directional keyboard input.
+//!     This is done in the `player` module, as it is specific to the player character.
```

Comment:
Well, unless we add `With<Player>` to the query, _technically_ it's not specific to the player character rn.

Reply:
[`ff2dbb3` (#236)](https://github.com/TheBevyFlock/bevy_quickstart/pull/236/commits/ff2dbb31f444d407c9a6f9442b0e91f7adec2f4f)